src/compiler/core.ts

```diff
@@ -633,6 +633,10 @@ module ts {
         }
     }
 
+    export function getDefaultLibraryFilename(options: CompilerOptions): string {
+        return options.target === ScriptTarget.ES6 ? "lib.es6.d.ts" : "lib.d.ts";
+    }
+

```

Comment:
@mhegazy I agree on that. Also, should we make sure that the target exist first? or we are guarantee to have default target as there was a problem in the harness when target isn't specified